website/project/views/contributor.py

```diff
@@ -378,6 +378,55 @@ def project_manage_contributors(auth, node, **kwargs):
     return {}
 
 
+@no_auto_transaction
+@must_be_valid_project  # injects project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def project_remove_contributor(auth, **kwargs):
```

Comment:
[Looks like](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/routes.py#L1267-L1270) `project_removecontributor` removes *all* contributors. Let's rename it, to reflect that and harmonize with the other view functions, to `project_remove_contributors`.